frontend/src/components/ParamsDefaultFields.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+  FormGroup,
+  FormHelperText,
+  HelperText,
+  HelperTextItem,
+  TextInput,
+} from '@patternfly/react-core';
+import * as React from 'react';
+import { JsonInputParam } from '~/concepts/pipelines/content/createRun/contentSections/ParamsSection/JsonInputParam';
+import { NumberInputParam } from '~/concepts/pipelines/content/createRun/contentSections/ParamsSection/NumberInputParam';
+import { RadioInputParam } from '~/concepts/pipelines/content/createRun/contentSections/ParamsSection/RadioInputParam';
+import {
+  InputDefinitionParameterType,
+  RuntimeConfigParamValue,
+} from '~/concepts/pipelines/kfTypes';
+
+type ParamsDefaultFieldsProps = {
+  parameterType: InputDefinitionParameterType | undefined;
+  inputProps: {
+    value: RuntimeConfigParamValue;
+    id: string;
+    name: string;
+    onChange: (
+      _event: React.ChangeEvent<unknown> | null,
+      newValue: string | number | boolean,
+    ) => unknown;
+  };
+  label: string;
+  description: string | undefined;
+  isOptional: boolean | undefined;
+  value: RuntimeConfigParamValue;
```

Comment:
you pass value twice. you dont need to.

you can just do this for the string field

```tsx
      input = <TextInput {...inputProps} value={String(inputProps.value)} />;
```